osu.Game/Beatmaps/Formats/IHasComboColours.cs

```diff
@@ -13,7 +11,7 @@ public interface IHasComboColours
         /// <summary>
         /// Retrieves the list of combo colours for presentation only.
         /// </summary>
-        IReadOnlyList<Color4> ComboColours { get; }
+        IReadOnlyList<Color4>? ComboColours { get; }
```

Comment:
This one's kinda painful, but can't be easily eliminated, so I'm looking away for the time being...

The long term answer to this is _probably_ interface separation or something like that.

Reply:
Yeah, right now it's there with reason so I didn't touch it.